docs/tutorials/advanced/using_the_cache.rst

```diff
@@ -0,0 +1,99 @@
+.. _using-the-cache:
+
+Using the cache
+===============
+
+Whilst for stochastic strategies every repetition of a Match will give a
+different result. For deterministic strategies, when there is no noise there is
```

Comment:
Whilst for stochastic strategies, every repetition of a Match will give a different result, for deterministic strategies, .....